packages/bitcore-node/test/unit/models/walletAddress.unit.ts

```diff
@@ -58,10 +58,12 @@ describe('WalletAddress Model', function () {
       expect(result.walletUpdates[0]).to.deep.equal({
         updateOne: {
           filter: {
+            // TODO:
```

Comment:
can you make this `// TODO: define wallet once wallet code is implemented`

Reply:
updated